app/src/main/java/org/wikipedia/history/db/HistoryEntryDao.kt

```diff
@@ -18,6 +18,9 @@ interface HistoryEntryDao {
     @Query("SELECT * FROM HistoryEntry WHERE authority = :authority AND lang = :lang AND apiTitle = :apiTitle LIMIT 1")
     suspend fun findEntryBy(authority: String, lang: String, apiTitle: String): HistoryEntry?
 
+    @Query("SELECT * FROM HistoryEntry ORDER BY random()  DESC LIMIT :limit")
```

Comment:
Extremely minor: double space after random(), and also, for maximum consistency, can we make it uppercase RANDOM()?

Reply:
Done!